mysql-test/suite/rocksdb/r/rocksdb.result

```diff
@@ -924,6 +924,7 @@ rocksdb_override_cf_options
 rocksdb_paranoid_checks	ON
 rocksdb_pause_background_work	ON
 rocksdb_perf_context_level	0
+rocksdb_persistent_cache_path	
```

Comment:
Also surprised rocksdb_persistent_cache_size is not showing up here in these test results. Do you need to re-record them?